<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemFrom</class>
 <widget class="QMainWindow" name="ItemFrom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>id</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="idLabel">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>姓名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="nameLabel">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>年龄</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="ageLabel">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>地址</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="addressLabel">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>薪水</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="salaryLabel">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
